content/how-to/dedicated-hosting/manage-local-users-in-the-myrackspace-portal/index.md

```diff
@@ -47,39 +55,39 @@ This article explains how to use a ticket template to manage local users.
       Administrators group on Windows&reg; and gives the user sudo privileges on
       Linux&reg;.)
 
-3. Select **Create Ticket**.
+**Step 3.** Click the **Submit** button.
 
-   {{<image src="localuser2.png" alt="" title="">}}
+   <img width="416" alt="Enter credential of new user" src="/support/how-to/manage-local-users-in-the-myrackspace-portal/localusermanagement6.png">
 
-#### Delete, Enable or Disable/lock a user
+**Delete, Enable or Disable/lock a User on the Selected Device**
 
-1. Select the desired action from the **Action Type** drop-down menu.
+**Step 1.** Select the desired action from the **Action Type** drop-down menu.
 
-2. Enter the username.
+**Step 2.** Enter the username.
 
    Deleting a user completely removes them from the selected devices.
 
-3. Select **Create Ticket**.
+**Step 3.** Click the **Submit** button.
 
-   {{<image src="localuser3.png" alt="" title="">}}
+   <img width="466" alt="Enter the name of the user to delete" src="/support/how-to/manage-local-users-in-the-myrackspace-portal/localusermanagement7.png">
 
-#### Change a user password
+**Change a User Password on the Selected Device**
 
-1. Specify the **Username** and **Password** to reset on the selected devices.
+**Step 1.** Enter the **Username** and **Password** to reset on the selected devices.
 
    The **Password** should be at least eight characters, with at least three
    character types (uppercase, lowercase, numeric, and symbol).
 
-2. Select **Create Ticket**.
+**Step 2.** Click the **Submit** button.
 
-   {{<image src="localuser4.png" alt="" title="">}}
+   <img width="440" alt="To change User password enter the suitable password" src="/support/how-to/manage-local-users-in-the-myrackspace-portal/localusermanagement8.png">
 
-#### Audit users or user groups  
+**Audit Users or User Groups on the Selected Device**  
 
 These actions provide a list of users or user groups on each selected server and
 require no additional input.
 
-{{<image src="localuser5.png" alt="" title="">}}
+  <img width="397" alt="Displays the list of users or user groups" src="/support/how-to/manage-local-users-in-the-myrackspace-portal/localusermanagement9.png">
```

Comment:
same question about data in the image. Is the device production data?

Reply:
Replace image as it was showing information about device.